openproblems/tasks/dimensional_reduction/README.md

```diff
@@ -5,7 +5,48 @@ Dimensional reduction is one of the key challenges in single-cell data represent
 
+Here, we add `adata.obsm`, which contains coordinates for various dimensional reduction methods. UMAP results are stored as `adata.obsm['X_umap']`, *t*-sne results as `adata.obsm['X_tsne']` and PCA results as `adata.obsm['X_pca']`.
+
+### CITE-seq dataset
+---
+We processed the CITE-seq dataset with ***Scanpy***, using a variety of dimensional reduction techniques offered within the package:
+<br><br><br>
+**PCA**
+![CITE-seq PCA](./images/pca_citeseq.png)
+<br><br>
+**UMAP**
+![CITE-seq UMAP](./images/umap_citeseq.png)
+<br><br>
+***t***-**SNE**
+![CITE-seq tSNE](./images/tsne_citeseq.png)
+<br><br>
+
+
+### The root mean square error
+---
+
+$$
+    RMSE = \sqrt{ \sum_{i=1}^{n} \frac{(\hat{y}_i - y_i)^2}{n} }
+$$
+
+Where $y_i$ is the sum of pairwise euclidian distances between each value embedded in low-dimensional space and $\hat{y_i}$ is the sum of pairwise euclidian distances between each value in the original, high-dimensional space. The goal, in terms of preservation of this space is to minimize the difference between these terms. Finding the root-mean of the square of all differences (Root mean square error or $RMSE$ is a simple way to represent this as a scalar, which can then be used to compare to other methods.  
+
+<a href = "http://cda.psych.uiuc.edu/psychometrika_highly_cited_articles/kruskal_1964a.pdf">Kruskel's stress</a> uses the RMSE, more or less in the now commonly-used MDS (multi-dimensional scaling). We can calculate and plot Kruskel's stress to get an idea where the majority of distortion of the ***topography*** of the data in high-dimensional space.
+
+![stress plot rmse CITE-seq](./images/stress_plot_citeseq_rmse.png)
+
+Above is a "complex heatmap", which aims to show the regions that contribute the most stress. You can see that while a majority of the stress comes from the left side of the plot (as shown by the top of the complex heat map), the center of that left set of clusters does not contribute much to the stress, leading us to believe that by the measure of RMSE, the topology is relatively well-preserved. The stress mostly comes from the clusters at the top and bottom of that group of clusters spread across the second PC. 
+
+
+![RMSE PCA CITE-seq](./images/rmse_citeseq.png)
+
+We performed principle component analysis, obtaining the first 50 components. We can then calculate the relative stress using the RMSE for each, in comparison to the original data, $y$. As one might suspect, the more components used, the lower the amount of distortion of the original data.
+
+![RMSE comparison](./images/rmse_comparison.png)
+
+We can make this comparison across multiple dimensionality reduction methods. We can see that *t*-SNE seems to distort the data the least, in terms of pairwise euclidian distances. This does not necessarily mean the data is best represented by *t*-SNE, however. There are multiple means of measuring the "goodness" of a dimensional reduction; RMSE is simply one of them.
```

Comment:
I lean towards not making any conclusions here about performance -- we'll leave that to the evaluation tables on the website.